examples/onnxrt/nlp/huggingface_model/text_generation/llama/quantization/weight_only/README.md

```diff
@@ -17,7 +17,7 @@ pip install -r requirements.txt
 ## 2. Prepare Model
 
 ```bash
-optimum-cli export onnx --model decapoda-research/llama-7b-hf --task text-generation-with-past ./llama_7b
+python prepare_model.py  --input_model="decapoda-research/llama-7b-hf" --output_model="./llama_7b"
```

Comment:
Unable to access decapoda-research/llama-7b-hf on huggingface

Reply:
fixed in this pr: https://github.com/intel/neural-compressor/pull/1345